src/components/Dashboard/TodoList.vue

```diff
@@ -713,24 +717,28 @@ export default {
           title: `File ${ARFilingYear} Annual Report`,
           subtitle,
           ARFilingYear,
-          arMinDate: todo.header.arMinDate, // COOP only
-          arMaxDate: todo.header.arMaxDate, // COOP only
-          status: todo.header.status || FilingStatus.NEW,
+          // NB: get min/max AR dates from header object (not business object)
+          // same as loading a draft AR
+          arMinDate: header.arMinDate, // COOP only
+          arMaxDate: header.arMaxDate, // COOP only
+          status: header.status || FilingStatus.NEW,
           enabled: Boolean(task.enabled),
           order: task.order,
           nextArDate: this.nextArDate(todo.business.nextAnnualReport), // BCOMP only
-          arDueDate: this.arDueDate(todo.business.nextAnnualReport) // BCOMP only
+          arDueDate: this.formatDateString(header.arMaxDate) // BCOMP only
```

Comment:
OK. Can we remove `todo.business.arMaxDate` and `todo.business.arMinDate` from the response from the legal API then?

Reply:
Yes. The UI does not use these.

(The API may still want to calculate these values to validate the AR date when the AR is filed.)

Do you have a ticket in mind already, or do we need a new ticket?

Reply:
@thorwolpert Is removing business.arMaxDate and business.arMinDate something you want to do in your ledger refactor work?

Reply:
> Do you have a ticket in mind already, or do we need a new ticket?

We don't have any ticket for that atm. That would be good.

Reply:
https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/8827